content/en/docs/concepts/verification-by-reference.md

```diff
@@ -306,21 +306,14 @@ Another index of type `notary.verification`, which adds the scanning verificatio
 }
 ```
 
-The nv2 client could retrieve all signatures with the following mocked command
+The notation client could retrieve all signatures with the following command
```

Comment:
This sentence can be removed since `notation pull` command is no longer supported